twig/BS3/forms/controller.twig

```diff
@@ -13,13 +13,12 @@
     {% endif %}
 {% endfor %}
 
-
 <div class="row">
     {% if FC_Config.getRenderArg('formArray')|length > 1 %}
         <div class="col-md-3">
             <div class="panel panel-sitespecific">
                 <div class="panel-heading">
-                    <div class="panel-title">Navigation</div>
+                    <{{ navHeading }} class="panel-title">Navigation</{{ navHeading }}>
```

Comment:
use `FC_Config.getNavHeading()` (or whatever the function name is)

Reply:
Changed!